soundconverter/interface/batch.py

```diff
@@ -352,6 +352,7 @@ def __init__(self, input_files):
             converter = Converter(sound_file, name_generator)
 
             converter.existing_behaviour = settings.get('existing')
+            converter.delete_original = settings.get('delete-original')
```

Comment:
Check [`use_memory_gsettings`, line 67](https://github.com/kassoulet/soundconverter/blob/main/soundconverter/interface/batch.py#L67). This should probably be stored there instead to be consistent with the other settings

Reply:
`existing_behaviour` seems to be only available via the batch mode. I don't remember why it is not a gio_setting as well, and it certainly causes some confusion.

Anyhow, I guess the current way of handling things if this doesn't change in that regard is: if the setting is relevant for both the gui and the batch mode, use `use_memory_gsettings`

Reply:
I did that in the first draft but isn't it persisted across restarts?